src/js/controllers/wizard.js

```diff
@@ -298,6 +298,10 @@ module.exports = function(app) {
         );
       };
 
+      $scope.isOnline = function () {
+        return window.navigator.onLine;
```

Comment:
The angular way is to use `$window` here and add it to the dependencies in the `app.controller(...)` line (analogous to `$http`). (Background info: makes testing easier because we're not accessing a global var and angular can provide a mocked window object).